resources/views/components/game/devbox-claim-management.blade.php

```diff
@@ -37,11 +36,11 @@
 $userHasClaimSlot = false;
 $openTickets = null;
 
+$userPermissions = $user?->getAttribute('Permissions') ?? 0;
```

Comment:
Recommend using enum as fallback value: `Permissions::Unregistered`

Reply:
Agreed, done in latest.